Mage.Sets/src/mage/cards/b/BluntTheAssault.java

```diff
@@ -18,7 +18,7 @@
  */
 public final class BluntTheAssault extends CardImpl {
 
-    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("creature on the battlefield");
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("creatures on the battlefield");
```

Comment:
This was correct before no?  "for each creature on the battlefield".

Reply:
In-game it reads "gains life equal to the number of creatures on the battlefield", previously it read "gains life equal to the number of **creature** on the battlefield", I just fixed the typo. Looking at the cards now, maybe setText should be used to get the "you gain 1 life for each.." text

Reply:
Hmm... looks like these life gain cards are kind of inconsistent on which wording they use.  Probably just use setText on these for now.